### DISCUSSION TITLE
Failed to run config for "nvim-snippets" after latest updated

### DISCUSSION THREAD

**Discussion by SargisPlusPlus** [LazyVim/LazyVim#3441] [General]
"nvim-snippets" started to fail after latest plugin install. Has anyone faced similar issue?

![Screenshot 2024-06-03 at 11 39 01 AM](https://github.com/LazyVim/LazyVim/assets/6013752/4fa6c9ca-02d3-4232-aa52-bde2addc3c7c)

**Comment by dpetka2001** [comment]
Do you have a `package.json` for your custom snippets? It's not needed any more. It's only taken into consideration for `friendly-snippets`.

**Reply by SargisPlusPlus** [reply]
no, I don't have a `package.json` for custom snippets 🤔

**Reply by dpetka2001** [reply]
I don't have such error even with the latest update of the plugin. What Neovim version are you using?

**Reply by SargisPlusPlus** [reply]
```
NVIM v0.9.4
Build type: Release
LuaJIT 2.1.1703358377

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/opt/homebrew/Cellar/neovim/0.9.4/share/nvim"```

**Reply by SargisPlusPlus** [reply]
Also, my latest configs are [here](https://github.com/SargisPlusPlus/nvim)

**Reply by dpetka2001** [reply]
https://github.com/garymjr/nvim-snippets?tab=readme-ov-file#requirements

**Reply by SargisPlusPlus** [reply]
Upgrading nvim to `0.10` fixed the issue. 
Thank you very much
